src/handlers/ground-truths/prompts.ts

```diff
@@ -0,0 +1,57 @@
+import { GroundTruthsSystemMessageTemplate, ModelApplications } from "../../types/llm";
+
+const CODE_REVIEW_GROUND_TRUTHS_SYSTEM_MESSAGE = {
+  example: [
+    `Using the input provided, your goal is to produce an array of strings that represent "Ground Truths."
+        These ground truths are high-level abstractions that encapsulate the key aspects of the task.
+        They serve to guide and inform our code review model's interpretation of the task by providing clear, concise, and explicit insights.
+        
+        Each ground truth should:
+        - Be succinct and easy to understand.
+        - Directly pertain to the task at hand.
+        - Focus on essential requirements, behaviors, or assumptions involved in the task.
+    
+        Example:
+        Task: Implement a function that adds two numbers.
+        Ground Truths:
+        - The function should accept two numerical inputs.
+        - The function should return the sum of the two inputs.
+        - Inputs must be validated to ensure they are numbers.
+        
+        Based on the given task, generate similar ground truths adhering to a maximum of 10.
+        
+        Return a JSON parsable array of strings representing the ground truths, without comment or directive.`,
+  ],
+  truthRules: [],
+  conditions: [],
+};
+
+const CHAT_BOT_GROUND_TRUTHS_SYSTEM_MESSAGE = {
+  truthRules: [
+    "Be succinct and easy to understand.",
+    "Use only the information provided in the input.",
+    "Focus on essential requirements, behaviors, or assumptions involved in the repository.",
+  ],
+  example: [
+    "Languages: { TypeScript: 60%, JavaScript: 15%, HTML: 10%, CSS: 5%, ... }",
+    "Dependencies: Esbuild, Wrangler, React, Tailwind CSS, ms, React-carousel, React-icons, ...",
+    "Dev Dependencies: @types/node, @types/jest, @mswjs, @testing-library/react, @testing-library/jest-dom, @Cypress ...",
+    "Ground Truths:",
+    "- The repo predominantly uses TypeScript, with JavaScript, HTML, and CSS also present.",
+    "- The repo is a React project that uses Tailwind CSS.",
+    "- The project is built with Esbuild and deployed with Wrangler, indicating a Cloudflare Workers project.",
+    "- The repo tests use Jest, Cypress, mswjs, and React Testing Library.",
+  ],
+  conditions: [
```

Comment:
Can you show me what you mean and convert one or two of those lines into what you mean exactly please bud? I don't quite understand as the condition is the list item

Reply:
```
1. ASSUME your output BUILDS the FOUNDATION for a chatbot to UNDERSTAND the REPOSITORY when asked an arbitrary QUERY.
2. DO NOT LIST every LANGUAGE or DEPENDENCY; FOCUS on the most PREVALENT ones.
3. FOCUS on what is ESSENTIAL to UNDERSTAND the REPOSITORY at a HIGH LEVEL.
4. BREVITY is KEY. USE NO FORMATTING — DO NOT WRAP in quotes, BACKTICKS, or other CHARACTERS.
5. EXPECTED FORMAT: response === ["some", "array", "of", "strings"].
```

Will that do? 

I'm aware capitalizing has it's benefits but haven't seen it used like this before, would be cool if you could link to the systems you are talking about.

I wouldn't have done this personally but if you think it'll be an improvement I will implement your approved conditions ^.

Reply:
I've used/seen it used sparingly throughout a prompt to enforce a nuance in formatting like `NEVER wrap in backticks` and that sort of thing or to really drive home one or two points but I do agree it produces a lot of value when used well.

Since this is a "test in production" sort of plugin, what I'll do is embed these ground truths into the final llm response and for when/if they are revisited later we'll have a good dataset to be able to refine the prompt with. (a set of truths is created with every query)

Reply:
OpenAI system prompts don't look like this so I'm inclined to believe that's not accurate.

Reply:
As am I but would you want me to still embed the truths that get sourced at least initially?

Also, we used to escape the OpenAI system message. We should probably bring that back.

Reply:
We can try ground truths I was referring to capitalized words for emphasis.

Reply:
I was referring to that also my question was referring to meaning using the SDK comment metadata feature